rclpy/src/rclpy/_rclpy.c

```diff
@@ -17,6 +17,7 @@
 #include <rmw/rmw.h>
 #include <rcl/error_handling.h>
 #include <rcl/rcl.h>
+#include <rcl/graph.h>
```

Comment:
Indeed linter missed it, https://github.com/ros2/rclpy/pull/40/commits/1c6942f3ddfec8fa8e95af9bf5441fb17cd6f625